### PR TITLE
Fixes #5031 Change the social icons border color

### LINKED ISSUE #5031: Change the social icons border color
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** . 
* The border and background of buttons in facebook and twitter buttons used is #4267b2 and #1b95e0 which is the main color of palette color followed by facebook and twitter. Let the background color remain same but Change the border hex to #29487d and ##00aced which is also a part of palette too but it shows a difference with background color.

Screenshot from 2019-03-12 22-55-25

*  update will be done in app/views/home/_social.html.erb

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by daddyjab**
Fixes #5031  (<=== Add issue number here)

Changed app/assets/stylesheets/dashboard.css to change color of the borders on the Facebook and Twitter buttons:

    Facebook button: Border changed from #4267b2 to #29487d
    Twitter button: Border changed from #1b95e0 to #00aced

NOTE: The issue created indicated that the changes should be made to views/home/_social.html.erb, but I instead made the change to the CSS file indicated above because it seemed more appropriate to control the button border color via the CSS. Also, I did not see a way to implement this border color change in the _social.html.erb file.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!
BEFORE-Button_Border_Colors_Match_Background_Color
BEFORE-Facebook_Button_Attribute_Change png
BEFORE-Twitter_Button_Attribute_Change
CODE_CHANGE-app_assets_stylesheets_dashboard_css
AFTER-Button_Border_Colors_Different_than_Background
AFTER-CSS_Change_Facebook
AFTER-CSS_Change_Twitter_NewerImage

### CODE REVIEW

**Comment by daddyjab** [comment]
@gauravano, @publiclab/reviewers:
Please review and accept this pull request.  Thanks! -- daddyjab

File: app/assets/stylesheets/dashboard.css [hunk 1]
```diff
@@ -216,6 +216,20 @@
   margin-right: 5px;
 }
 
+/* Issue #5031 (daddyjab):
```
Comment (ananyaarun) [inline app/assets/stylesheets/dashboard.css]: Hey @daddyjab  can u remove the comments , Thanks !!

File: app/assets/stylesheets/dashboard.css [hunk 2]
```diff
@@ -216,6 +216,20 @@
...
+   Revised to add border of
+   offsetting color to Twitter buttons */
+.dashboard .social-media-btn #follow-button {
+  border: 1px solid #00aced;
+}
+
+/* Issue #5031 (daddyjab):
```
Comment (ananyaarun) [inline app/assets/stylesheets/dashboard.css]: Here as well :) 
Comments are always nice to understand the code but i feel it is redundant here !

**Comment by ananyaarun** [comment]
Also i think it's better to change the PR name to just ''Changed the social icon border color'' :) Thanks.

**Comment by daddyjab** [comment]
Hi @ananyaarun
I had an initial pull request that included thr comments, but after feedback on that I closed that pull request without merge and opened a subsequent pull request addressing this and other feedback, including an error I had in the first change.

Please see pull request 5425 on this thread, which I submitted with title "Fixes #5031 Change the social icons border color"

Thanks! - daddyjab

**Comment by ananyaarun** [comment]
Oh, i saw that now :) Good work.
Btw have you been able to set up public labs locally on your system ?
You can follow the instructions on readme if u haven't, so that next time u can give screenshots  of the changes from ur local installations instead of using inspect element to make changes.
Thanks !!

**Comment by gauravano** [comment]
@ananyaarun @CleverFool77 you want to leave a review here? Thanks!

**Comment by daddyjab** [comment]
Thank you @cleverfool77, @gauravano, and @ananyaarun for your support!

**Comment by gauravano** [comment]
I tried this PR locally but I am not able to see any changes. @daddyjab have you installed plots2?  If yes, can you give it a try there? Thanks!

**Comment by SidharthBansal** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by SidharthBansal** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by gauravano** [comment]
hey @CleverFool77, can you try this PR locally?

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Pull request closed without merging**